pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -233,8 +233,13 @@ def connection_save(context, name):
 
 @connection_group.command('list', cls=PywbemcliCommand,
                           options_metavar=CMD_OPTS_TXT)
+@click.option('-f', '--full', is_flag=True,
+              default=False,
+              help='If set, display the full table. Otherwise display '
+                   ' a brief view(name, server, mock_server columns). '
+                   'This table does not show the ca_certs field.')
```

Comment:
To me, "This table" at this point sounds ambiguous, it could be the brief view the previous sentence talks about, or the full table.

Reply:
Removed the whole sentence.

Reply:
DONE